tools/distpartitioning/utils.py

```diff
@@ -377,6 +377,12 @@ def write_edge_features(edge_features, edge_file):
     edge_file     : string 
         File in which the edge information is serialized
     """
+    def format_etype(etype):
+        etype, name = etype.split('/')
+        etype = etype.split(':')[1]
+        return etype + '/' + name
```

Comment:
I generally don't like this kind of variable shadowing. Making it difficult to read. Also, is this to convert canonical etype (in the form of `'type_1:relation:type_2'`) to etype? Is this temporary as we are shifting to canonical etype entirely?

Reply:
Sure. Let's fix them in one PR that completely drops the support for string etype.